js/contact.js

```diff
@@ -2,7 +2,7 @@ async function doDeleteContact() {
   if (confirm("Are you sure you want to delete this contact?")) {
     const contact = JSON.parse(localStorage.getItem('individualContact'));
     const [status, responseJson] = await postData(
```

Comment:
```suggestion
    const [status, responseJson] = await postData(
```
For the Delete function you will have to create another await helper function "deleteData" which will have request method set to "DELETE"